Runtime/Haptics/AudioClipHapticPulser.cs

```diff
@@ -41,13 +46,13 @@ public class AudioClipHapticPulser : HapticProcess
         /// <inheritdoc />
         public override bool IsActive()
         {
-            return (base.IsActive() && hapticPulser != null && audioClip != null && hapticPulser.IsActive());
+            return (base.IsActive() && HapticPulser != null && AudioClip != null && HapticPulser.IsActive());
```

Comment:
```suggestion
            return base.IsActive() && HapticPulser != null && AudioClip != null && HapticPulser.IsActive();
```
Might as well while we're touching this. 😄